docs/examples/configuration/test_example_8.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from uuid import UUID
+
+import pytest
+
+from polyfactory import ConfigurationException, PostGenerated
+from polyfactory.factories.dataclass_factory import DataclassFactory
+
+
+@dataclass
+class Person:
+    id: UUID
+
+
+with pytest.raises(
+    ConfigurationException,
+    match="unknown_field is declared on the factory PersonFactory but it is not part of the model Person",
+):
+
+    class PersonFactory(DataclassFactory[Person]):
+        __model__ = Person
+        __check_model__ = True
+        unknown_field = PostGenerated(lambda: "foo")
```

Comment:
This should go into a `test_<some_descriptive_name>` function so that `pytest` picks it up as a test when testing the examples.

Reply:
Well it's picked up because it's imported but I can wrap it in a function if you like it better that way :wink:

Reply:
Yeah it does get picked up, but then if it fails, then we don't get a nice error output from `pytest`. It'll fail even before the other tests are run since it'll fail during the test collection phase itself.